node/reqres.js

```diff
@@ -76,8 +77,10 @@ function TChannelOutgoingRequest(id, options, sendFrame) {
     self.id = id || 0;
     self.ttl = options.ttl || 0;
     self.tracing = options.tracing || emptyTracing;
+    self.tracer = options.tracer; // tracing agent
```

Comment:
dangerous: a shared mutable tracer between multiple ongoing req / res objects in the same process (later mutated by `tracer.setFoo` methods)